data/JenilGajjar20.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "Jenil Gajjar",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Frontend Developer 👨‍💻 || Self-learner || Eager to learn new technologies || Open Source ❤ || I write articles on Medium",
+  "avatar": "https://github.com/JenilGajjar20.png",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/gajjar_jenil",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/jenil-gajjar-27934920a/",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/JenilGajjar20",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "Follow me on Medium",
+      "url": "https://jenilgajjar.medium.com/",
+      "icon": "FaMedium"
+    },
```

Comment:
hey this comma at the end of line 27 needs to be removed

Reply:
Okay, change Updated